androidApp/src/main/java/dev/johnoreilly/confetti/sessions/SessionListView.kt

```diff
@@ -129,6 +130,7 @@ fun SessionListView(
     }
 }
 
+private val dateFormatter = DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM)
```

Comment:
Also, if you prefer a different format let me know. The `MEDIUM` seems like a good one IMO.